Resources/Public/Xsd/dfg-viewer.xsd

```diff
@@ -0,0 +1,309 @@
+          <xs:annotation>
+            <xs:documentation>logo URL of the institution holding the digital object</xs:documentation>
+          </xs:annotation>
+        </xs:element>
+        <xs:element name="ownerSiteURL" type="xs:anyURI">
+          <xs:annotation>
+            <xs:documentation>website URL of the institution holding the digital object</xs:documentation>
+          </xs:annotation>
+        </xs:element>
+        <xs:element name="ownerContact" type="xs:anyURI">
+          <xs:annotation>
+            <xs:documentation>contact (email/webform) at the institution holding the digital object</xs:documentation>
+          </xs:annotation>
+        </xs:element>
+        <xs:element name="aggregator" type="xs:string" minOccurs="0">
+          <xs:annotation>
+            <xs:documentation>name of the aggregator or portal owning the data to the digital object</xs:documentation>
+          </xs:annotation>
+        </xs:element>
+        <xs:element name="aggregatorLogo" type="xs:anyURI" minOccurs="0">
+          <xs:annotation>
+            <xs:documentation>logo URL of the aggregator or portal owning the data to the digital object</xs:documentation>
+          </xs:annotation>
+        </xs:element>
+        <xs:element name="aggregatorSiteURL" type="xs:anyURI" minOccurs="0">
+          <xs:annotation>
+            <xs:documentation>website URL of the aggregator or portal owning the data to the digital object</xs:documentation>
+          </xs:annotation>
+        </xs:element>
+        <xs:element name="aggregatorContact" type="xs:anyURI" minOccurs="0">
+          <xs:annotation>
+            <xs:documentation>contact (email/webform) at the aggregator or portal owning the data to the digital object</xs:documentation>
+          </xs:annotation>
+        </xs:element>
+        <xs:element name="sponsor" type="xs:string" minOccurs="0">
+          <xs:annotation>
+            <xs:documentation>name of the sponsor for the digitization process</xs:documentation>
+          </xs:annotation>
+        </xs:element>
+        <xs:element name="sponsorLogo" type="xs:anyURI" minOccurs="0">
+          <xs:annotation>
+            <xs:documentation>logo URL of the sponsor for the digitization process</xs:documentation>
+          </xs:annotation>
+        </xs:element>
+        <xs:element name="sponsorSiteURL" type="xs:anyURI" minOccurs="0">
+          <xs:annotation>
+            <xs:documentation>website URL of the sponsor for the digitization process</xs:documentation>
+          </xs:annotation>
+        </xs:element>
+        <xs:element name="sponsorContact" type="xs:anyURI" minOccurs="0">
+          <xs:annotation>
+            <xs:documentation>contact (email/webform) at the sponsor for the digitization process</xs:documentation>
+          </xs:annotation>
+        </xs:element>
+        <xs:element name="license" minOccurs="0" default="reserved">
+          <xs:simpleType>
+            <xs:restriction base="xs:string">
```

Comment:
Or do we have to include other values (like CNE or NoC) here, too?

Cf. [Europeana license codes](https://europeana.atlassian.net/wiki/spaces/EF/pages/1503756289/Providing+copyright+metadata+to+Europeana).

Reply:
See [respective DDB schematron rule](https://github.com/Deutsche-Digitale-Bibliothek/ddb-metadata-schematron-validation/blob/631fcdd6092fc1d0621501857ae3c1aac0afa96b/mets-mods-ap-digitalisierte-medien/ddb_validierung_mets-mods-ap-digitalisierte-medien.sch#L1053-L1064)